bin/send_lhep8.py

```diff
@@ -106,13 +111,29 @@ def send(self, force):
             if self.decay not in self.para.decaylist[pr_noht]:
                 print 'decay ==%s== does not exist for process ==%s=='%(self.decay,self.process)
                 sys.exit(3)
+        '''
+        ##FIXME have to make py card a mandatory argument
```

Comment:
this is done or?

Reply:
yes, not needed since default pythia_card specified

Reply:
added instruction in README